src/Services/FileManagement/FileManagementService.php

```diff
@@ -237,4 +248,101 @@ public function getErrors($e)
             }
         }
     }
+
+    /**
+     * Create File variant
+     *
+     * @param string $uuid
+     * @param string $pluginCode
+     *
+     * @return bool
+     *
+     */
+    public function createFileVariant($uuid, $pluginCode)
+    {
+        try {
+            Log::info("CREATING FILE VARIANTS:");
+
+            Log::info("Saving variant details in database...");
+            $data = new FileManagementVariantLogData($uuid);
+            $data->setVariantType($pluginCode);
+            $this->fileManagementVariantLogRepository->store($data);
+
+            $response = $this->makeRequest(
+                Constants::$POST_METHOD,
+                collect([
+                    "uuid" => $uuid,
+                    "plugins" => [collect([
+                        "code" => $pluginCode
+                    ])]
+                ]),
+                env("FILE_MANAGEMENT_URL").Endpoints::$FILE_VARIANTS,
+                $this->getFileMangementHeader(),
+                []
+            );
```

Comment:
This API also returns the variant id in the response, you can update the variant id here

Reply:
Okay.. but I have updated the variant ID in the controller itself.

Reply:
This function will run first, update the variant id here